plugins/linux-pipewire/portal.c

```diff
@@ -73,6 +73,25 @@ uint32_t portal_get_available_capture_types(void)
 	return available_source_types;
 }
 
+enum portal_cursor_mode portal_get_available_cursor_modes(void)
+{
+	g_autoptr(GVariant) cached_cursor_modes = NULL;
+	uint32_t available_cursor_modes;
+
+	ensure_proxy();
+
+	if (!proxy)
+		return 0;
```

Comment:
Should this be `0` or should `0` be represented by an enum entry or define?

Reply:
I'd say it should be zero, because this code path should return something to represent "couldn't fetch", which is a state we shouldn't reach when things are fine. This is different from the HIDDEN state, which is set to 1.

Reply:
I guess my question was more, "Should we explicitly add an entry for `0` in the enum so that it's descriptive?" I'm fine either way.

Reply:
Sorry, I was *trying* to answer this question :slightly_smiling_face: 

My usual stance on enums is that there needs to be a very specific reason to enumerate "invalid" - for example, when the "invalid" value is passed as actual, expected input somewhere. This is not the case here. In the case of this code, the invalid case is just something we have to loosely protect against, so I don't see any reason to upcast it into the enum.

Reply:
Good enough for me.